src/bulk_insert/bulk_insert.c

```diff
@@ -68,8 +68,6 @@ static inline SIValue _BulkInsert_ReadProperty(const char *data, size_t *data_id
 	TYPE t = data[*data_idx];
 	*data_idx += 1;
 	if(t == BI_NULL) {
-		// TODO This property will currently get entered with a NULL key.
-		// Update so that the entire key-value pair is omitted from the node
 		v = SI_NullVal();
```

Comment:
if we are skipping null properties it in `BulkInsert_ProcessNodeFile`, we should be here, right?

Reply:
This function invocation (`_BulkInsert_ReadProperty`) is directly before the null check.
I prefer the ordering in this PR since the inputs to `_BulkInsert_ReadProperty` are a byte stream - we could check the stream before this call, but I think it would be worse abstraction-wise.